conductor/blueprints/authorization/controllers.py

```diff
@@ -69,21 +72,9 @@ def __call__(self):
         return PUBLIC_KEY
 
 
-class Sample:
```

Comment:
I think that all controllers in this module are stateless, so we can probably move them all to be functions.
@pwalsh is correct, I kept the style of the original author.
I opened #7 which will contain a complete re-styling of the code based on our current style guide and tooling.